src/web/browser/App.tsx

```diff
@@ -124,13 +125,18 @@ const App = ({ CAPI, NAV }: Props) => {
             <Hydrate root="nav-root">
                 <Nav pillar={CAPI.pillar} nav={NAV} />
             </Hydrate>
-            <Hydrate root="sub-nav-root">
-                <SubNav
-                    subnav={NAV.subNavSections}
-                    currentNavLink={NAV.currentNavLink}
-                    pillar={CAPI.pillar}
-                />
-            </Hydrate>
+            {NAV.subNavSections && (
+                <Hydrate root="sub-nav-root">
+                    <>
+                        <SubNav
+                            subNavSections={NAV.subNavSections}
+                            currentNavLink={NAV.currentNavLink}
+                            pillar={CAPI.pillar}
+                        />
+                        <GuardianLines pillar={CAPI.pillar} />
```

Comment:
These don't need hydrating - we only need to server side render them and move on